packages/blade/src/components/List/ListItem.tsx

```diff
@@ -60,6 +61,36 @@ const StyledListItem = styled(ListItemElement)<{
     : 0,
 }));
 
+const ListItemContentChildren = ({
+  children,
+  size,
+}: {
+  children: (React.ReactChild | React.ReactFragment | React.ReactPortal)[];
```

Comment:
Any reason we kept this so specific? eg: Fragment, ReactPortal. 

Can we use React.ReactNode instead?

Reply:
Had done it to suppress a warning. Made it ReactNode now. Draft PR hai, have mercy 😢

Reply:
lol sorry, was reviewing this at airport :p